index.html

```diff
@@ -592,6 +592,7 @@
 			var originalZoomLevel = map.getZoom();
 
 			var loadedPointLayers = [];
+			var loadedPointLayerNames = [];
```

Comment:
This looks duplicative, but it's because `loadedPointLayers` will store an array of `[layerName, legendID]` pairs, which is used elsewhere to activate all layers on zoom, where `loadedPointLayerNames` is an array of just the layer names.  That in turn gets used in the click handler to check for whether any point has been clicked on.